test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -234,35 +237,14 @@ public void CanReadWorksheetWithEmptyCellInUsedRange()
             ViewModel.OpenCommand.Execute(openPath);
 
             var filename = ViewModel.Model.CurrentWorkspace.FirstNodeFromWorkspace<DSCore.File.Filename>();
-
             // remap the filename as Excel requires an absolute path
             filename.Value = filename.Value.Replace(@"..\..\..\test", TestDirectory);
-
             var watch = ViewModel.Model.CurrentWorkspace.GetDSFunctionNodeFromWorkspace("Excel.GetDataFromExcelWorksheet");
-
             ViewModel.HomeSpace.Run();
-
-            Assert.IsTrue(watch.CachedValue.IsCollection);
+            //Assert.IsTrue(watch.CachedValue.IsCollection);
             var list = watch.CachedValue.GetElements();
-
-            Assert.AreEqual(4, list.Count);
-
-            // single column - 1, "word", 2, 3, "palabra"
-            Assert.IsTrue(list[0].IsCollection);
-            var rowList = list[0].GetElements();
-            Assert.AreEqual("a", rowList[0].Data);
-
-            Assert.IsTrue(list[1].IsCollection);
-            rowList = list[1].GetElements();
-            Assert.IsNull(rowList[0].Data);
-
-            Assert.IsTrue(list[2].IsCollection);
-            rowList = list[2].GetElements();
-            Assert.AreEqual("cell is", rowList[0].Data);
-
-            Assert.IsTrue(list[3].IsCollection);
-            rowList = list[3].GetElements();
-            Assert.AreEqual("missing", rowList[0].Data);
+            var data = new object[] {new object []{"a"},new object []{null},new object []{"cell is"},new object[] {"missing"} };
```

Comment:
It would be good to put a comment to show the data //data = {{"a"}, {null}, {"cell is"}, {"missing"}}
You can simplify your verification code, by using a transpose node in the graph and verify the preview value of transpose node.